hawc/apps/common/views.py

```diff
@@ -704,19 +704,23 @@ def get_paginate_by(self, qs) -> int:
     def get_filterset_kwargs(self):
         return dict(
             data=self.request.GET,
-            queryset=super().get_queryset(),
+            queryset=self.get_base_queryset(),
             request=self.request,
+            form_kwargs=self.get_form_options(),
         )
 
-    def get_filterset_class(self) -> type[BaseFilterSet]:
-        return self.filterset_class
+    def get_form_options(self):
```

Comment:
Rename `get_form_kwargs`, or `get_filterset_form_kwargs`? It'd be more consistent with django ccbv ? I dont think we ever use this mixin on a formview, so it might be nice for consistency

https://ccbv.co.uk/projects/Django/4.2/django.views.generic.edit/FormView/